builds/flu/flu_titers.py

```diff
@@ -48,10 +49,73 @@ def HI_model(process):
         node.attr["cTiterSub"] = node.cTiterSub
         node.attr["dTiterSub"] = node.dTiterSub
 
+def vaccine_distance(process, attributes=['ep', 'dTiter', 'dTiterSub']):
+    '''
+    For each tip in the tree, trace the path between the tip and each vaccine strain.
+    Sum up the requested attributes along this path.
+    Also pull the tip's strain name, numdate, and region.
+
+    Writes to prefix_vaccine_dist.json
+    e.g.,
+        [
+      {
+        "strain": "A/Incheon/677/2006",
+        "num_date": 2006.867,
+        "region": "japan_korea",
+        "ep": {
+          "A/Perth/16/2009": 4,
+          "A/Victoria/361/2011": 6
+        },
+        "cTiter": {
+          "A/Perth/16/2009": 2.3,
+          "A/Victoria/361/2011": 4.2
+        },
+        "cTiterSub": {
+          "A/Perth/16/2009": 1.9,
+          "A/Victoria/361/2011": 3.7
+        }
+      }
+    ]
+    '''
+
+
+    tree = process.tree.tree
+    tips = [ k for k in tree.get_terminals() ] # all tips in the tree
+    vaccine_strains = vaccine_choices[process.info['lineage']]  # list of strains
+    vaccine_tips = { k.name: k for k in tips if k.name in vaccine_strains } # { strain_name: biophylo object}
+
+    def sum_along_path(path, attribute):
```

Comment:
This is a minor style preference, but I would move this function out into the top-level module, so it can be reused.